templates/configuration/graphdb-properties-configmap.yaml

```diff
@@ -1,17 +1,16 @@
 # Default configuration map for provisioning GraphDB repository.
```

Comment:
wrong comment btw, I've probably copied it from the repo config. Can you fix it? As well as the comments below for the settings configmap

Reply:
Fixed for all 5 configmaps that contain the config files.